apps/src/music/Timeline.jsx

```diff
@@ -142,7 +166,7 @@ export default class Timeline extends React.Component {
                       'solid 2px ' +
                       this.getColorsForEventId(eventData.id).border,
                     borderRadius: 8,
-                    height: 18
+                    height: this.getEventHeight() - 6
```

Comment:
nit: constant for the `6` just so it doesn't feel like a magic number?

Reply:
Good call.  Also adjusted the box sizing so that we don't need to include the border in this number.